contracts/bridge/BridgeTokens.sol

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.5.6;
+
+import "../externals/openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract BridgeTokens is Ownable {
+    mapping(address => address) public registeredTokens; // <token, counterpart token>
+    mapping(address => uint) public indexOfTokens; // <token, index>
+    address[] public registeredTokenList;
```

Comment:
Why do you need `registeredTokenList` at all?
It seems like its used only for read-only purpose, via function `getRegisteredTokenList`.
If that is the only case, you can make recover the full list without `getRegisteredTokenList` by reading trough the events. 
Using two types of storage (index mapping&list) for a read-only purpose seems a bit costly

Reply:
A service provider requested the feature to get the full list of registered token.
Therefore I added this feature. Also, without this list, it is hard to get the list.
As you said, through events, we can get it. But we should track all blocks. I think it is also costly.

Reply:
Alright. but FYR, if you use events in linked list form, you don't have to sweep the whole blocks. Since using storage is **the most costly** operation on blockchain, for most cases you better avoid using it, unless you need on chain calculation.

Reply:
As we talked in slack, the token `registeredTokenList` is for getting the list easily not saving storage. :) I will keep your idea for other storage issue. Thanks. =)